probatus/stat_tests/distribution_statistics.py

```diff
@@ -339,11 +340,16 @@ def compute(
         ):
             if self.binning_strategies == ["default"]:
                 bin_strat = DistributionStatistics.statistical_test_dict[stat_test]["default_binning"]
+            
             dist = DistributionStatistics(statistical_test=stat_test, binning_strategy=bin_strat, bin_count=bins)
             try:
                 if suppress_warnings:
                     warnings.filterwarnings("ignore")
-                _ = dist.compute(df1[col], df2[col])
+                # Issue a warning if missing values are present in one of the two columns. These observations are removed
+                # in the calculations. 
+                if np.sum(df1[col].isna()) + np.sum(df2[col].isna()):
+                    warnings.warn(f"Missing values in column {col+1} have been removed")
```

Comment:
Why is it col+1? Shouldn't it be col? 

I think also col might be a string right?

Reply:
Correct. I have change it in the last push